content/en/t/_index.md

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "Tracker",
```

Comment:
```suggestion
  "title": "Tracker documentation",
```

Reply:
Or, I guess, "Trackers" if you don't like that. But "Tracker" just sounds weird as a heading.